packages/salesforcedx-vscode-apex-debugger/package.json

```diff
@@ -83,7 +83,8 @@
     "packageUpdates": {
       "main": "dist/index.js",
       "dependencies": {
-        "applicationinsights": "1.0.7"
+        "applicationinsights": "1.0.7",
+        "@salesforce/core": "3.30.9"
```

Comment:
Just to confirm, we want this to be pinned and not carroted?

Reply:
correct. We want all the extensions to stay consistent 👍 good q though